src/app/components/CourseChoice/CourseChoice.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import styles from "./CourseChoice.module.css";
-import BinIcon from "../../assets/icons/BinIcon";
 import PrimPrioBall from "../../assets/icons/PrimPrioBall";
 import SecPrioBall from "../../assets/icons/SecPrioBall";
 import ThirdPrioBall from "../../assets/icons/ThirdPrioBall";
+import CourseChoiceChosen from "../CourseChoiceChosen/CourseChoiceChosen";
+import CourseChoiceButton from "../CourseChoiceButton/CourseChoiceButton";
```

Comment:
Isn't this a link and not a button? I'm not sure, but I had some problems myself with the differentiation in my app 😬

Reply:
Yes. A link! I changed it. Helpful! Thank you!